docs/content/concepts.mdx

```diff
@@ -131,86 +71,134 @@ A software-defined asset or job can represent a collection of _partitions_ that
 
 ---
 
-## I/O management
+## Resources & configuration
+
+Resources enable you to separate logic from external dependencies, making developing and testing possible in multiple environments. [Learn more about resources](/concepts/resources).
```

Comment:
how does this look if the Resources page is a bullet like all of the other sub sections? It was a tiny bit strange to see the link for the resources page as part of a sentence when all of the other links are bullets

Reply:
or maybe just having the "Learn more about resources" be on a new line like the Dagster UI section